backend/src/packages/business/business.entity.ts

```diff
@@ -12,42 +12,59 @@ class BusinessEntity implements IEntity {
 
   private ownerId: BusinessEntityT['ownerId'];
 
+  private stripeId: BusinessEntityT['stripeId'] | null;
+
+  private stripeActivated: BusinessEntityT['stripeActivated'];
```

Comment:
I know it sounds good and semantically right but [we have quality criteria B19](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md)